api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -26,6 +25,15 @@
     <handler file="server_unavailable.html" />
   </static-error-handlers>
 
+  <static-files>
+    <include path="static/activity-checker-extension.js" />
```

Comment:
I just remembered we should probably include this set of http headers: https://github.com/all-of-us/workbench/blob/e70ef354fe09414f273058d6c805e1bcc3212ab5/ui/app.yaml#L11-L15

This would probably push us towards pulling in the whole static/ directory to avoid repeating these on every include element.